src/Controller/GUIController.py

```diff
@@ -7,11 +7,38 @@
 from src.Model.PatientDictContainer import PatientDictContainer
 from src.View.OpenPatientWindow import UIOpenPatientWindow
 from src.View.PyradiProgressBar import PyradiExtended
+from src.View.FirstTimeWelcomeWindow import UIFirstTimeWelcomeWindow
 from src.View.WelcomeWindow import UIWelcomeWindow
 from src.View.mainpage.MainPage import UIMainWindow
 from src.Controller.PathHandler import resource_path
 
 
+class FirstTimeWelcomeWindow(QtWidgets.QMainWindow, UIFirstTimeWelcomeWindow):
+
+    update_directory = QtCore.Signal(str)
+    go_next_window = QtCore.Signal()
+
+    # Initialisation function to display the UI
```

Comment:
Ask if they wanted it as an s or z

Reply:
I would not worry about the Queen's vs American spelling in a comment.
For text display to the user, this is an Ozzie driven effort, so I think people should be OK with it being Queen's English.  I am at the point where I just let the spell checker choose.  My native spelling is American but at this point I usually aim at the Queen's English spelling from muscle memory